templates/etc/sensu/uchiwa.json.erb

```diff
@@ -47,6 +47,12 @@
     "auth": {
       "publickey": "<%= @auth['publickey']  %>",
       "privatekey": "<%= @auth['privatekey']  %>"
+    }<%= ',' if @ssl.size == 2 %>
```

Comment:
I don't think thats possible. The ',' should come after the '}' according to JSON standards.
I just copied the same style from the auth section.